src/StatsGenerator.ts

```diff
@@ -90,6 +100,41 @@ export class StatsGenerator {
         }
     }
 
+    public addPlayer(serverId: string, username: string) : boolean {
+        const server = this.getServer(serverId);
+        if (server){
+            const player = {SR: 0, player: username, private: false} as Player;
+            server.players.push(player);
```

Comment:
I think I started this fire but we gotta stop mutating the lastresult object (which `getServer` returns).  I'm ok with it in this PR but flagging we need to tech debt a proper implementation to update server properties

Reply:
Trusty ole tech debt